src/gretel_synthetics/config.py

```diff
@@ -135,7 +154,7 @@ class BaseConfig:
 
     # Diff privacy configs
     dp: bool = False
-    dp_learning_rate: float = 0.015
+    dp_learning_rate: float = 0.001
```

Comment:
Let's use 0.01 as a setting, which matches the hard-coded learning rate for non-DP operation w/RMSprop. Will update as we optimize hyper params and optimizers in 15.x